backend/src/packages/genres/genre.entity.ts

```diff
@@ -0,0 +1,57 @@
+import { type IEntity } from '~/libs/interfaces/interfaces.js';
+
+class GenreEntity implements IEntity {
+  private 'id': number;
+
+  private 'name': string;
+
+  private 'key': string;
+
+  private constructor({
+    id,
+    name,
+    key,
+  }: {
+    id: number;
+    name: string;
+    key: string;
+  }) {
```

Comment:
Please, create a type for it and put it right in this file (right before GenreEntity)

Reply:
Also, check all other places, where this case with type is not covered (not just method parameters but return types as well)

Reply:
Do you mean in other entities too?

Reply:
@WilhelmWesser i think we use the same approach in all entities, so there is no need to change it in this pr